tests/integrationv2/test_client_authentication.py

```diff
@@ -4,10 +4,10 @@
 from configuration import (available_ports, ALL_TEST_CIPHERS, PROTOCOLS)
 from common import Certificates, ProviderOptions, Protocols, data_bytes
 from fixtures import managed_process  # lgtm [py/unused-import]
-from providers import Provider, S2N, OpenSSL
+from global_flags import S2N_PROVIDER_VERSION, get_flag
+from providers import Provider, S2N, OpenSSL, GnuTLS
```

Comment:
```suggestion
from providers import Provider, S2N, GnuTLS
```

Reply:
Import is required since `OpenSSL` is a provider in the other tests.

Reply:
Whoops, sorry, I misread this.